_data-prepper/pipelines/configuration/sources/dynamo-db.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: dynamodb
+parent: Sources
+grand_parent: Pipelines
+nav_order: 3
+---
+
+# dynamodb
+
+The `dynamodb` source enables change data capture (CDC) which streams [DynamoDB](https://aws.amazon.com/dynamodb/) table events such as `create`, `update`, or `delete` to Data Prepper events in near real-time.
+
+The source includes two ingestion options to stream DynamoDB events:
+
+1. _Full Initial Snapshot_ during [point-in-time recovery (PITR)](https://aws.amazon.com/dynamodb/pitr/) which backs your table automatically. This requires both the PITR Snapshots and DyanmoDB streaming options are enabled on your DynamoDB table.
+2.  Stream events from DynamoDB streams without full initial snapshots. This is useful for users who already have a mechanism for snapshots within their pipelines. This requires that DynamoDB stream is enabled on the DynamoDB table.
+
+## Usage
+
+The following example pipeline specifies DynamoDB as a source. It ingests data from a DyanmoDB table, named `table-a` through a PITR snapshot. It also indicates the `start_position`, which tells the pipeline how to read events DynamoDB stream. The sink wi
+
+```json
+version: "2"
+cdc-pipeline:
+  source:
+    dynamodb:
+      tables:
+        - table_arn: "arn:aws:dynamodb:us-west-2:123456789012:table/table-a"
+          export:
+            s3_bucket: "test-bucket"
+            s3_prefix: "xxx/"
+          stream:
+            start_position: "LATEST" # Read latest data from streams (Default)
+      aws:
+        region: "us-west-2"
+        sts_role_arn: "role-arn"
+```
+
+## Configuration options
+
+The following tables describe the configuration options for the `dynamodb` source.
+
+### Shared options
+
+The following settings are shared between the `dynamodb` source and your data stores.
+
+Option | Required | Type | Description
```

Comment:
We need the following options:

* acknowledgments (see the `s3` source and use the same basic wording)
* shard_acknowledgment_timeout
* s3_data_file_acknowledgment_timeout
* tables - This will also need a section below.

Reply:
Do you have descriptions for the middle two?